modules/gapi/test/common/gapi_render_tests.hpp

```diff
@@ -78,7 +78,7 @@ class RenderWithParam : public TestWithParam<T>
     cv::Mat y_mat_ocv, uv_mat_ocv, y_mat_gapi, uv_mat_gapi, mat_ocv, mat_gapi;
 };
 
-using TestArgs = std::tuple<cv::Size,cv::gapi::wip::draw::Prims,cv::gapi::GKernelPackage>;
+using TestArgs = std::tuple<cv::Size,cv::gapi::wip::draw::Prims>;
```

Comment:
I think there is no reason to add `GCompileArgs` to test parameters because we have only `use_freetype` but not testing it on CI. Maybe later.

Reply:
But how do we instantiate these tests with proper backend then?

Reply:
I will add this in PR about integrate freetype

Reply:
- But how do we instantiate these tests with proper backend then?"
- It's not necessary we have a single render backend (I'm going to refactor tests in the near future)